tests/pykafka/test_balancedconsumer.py

```diff
@@ -56,18 +55,18 @@ def test_consume_graceful_stop(self):
     def test_decide_partitions(self):
         """Test partition assignment for a number of partitions/consumers."""
         # 100 test iterations
-        for i in xrange(100):
+        for i in range(100):
             # Set up partitions, cluster, etc
             num_participants = i + 1
             num_partitions = 100 - i
-            participants = ['test-debian:{p}'.format(p=p)
-                            for p in xrange(num_participants)]
+            participants = sorted(['test-debian:{p}'.format(p=p)
```

Comment:
Since the `_decide_partitions` function used `.sort()` instead of `sorted`, it was modifying the test function's copy of the `participants` array. This made the `idx` assignment always match its value in `_decide_partitions`, which could potentially have masked some errors. This way is actually better.